### ISSUE TITLE
Custom Style Not Applied

### ISSUE THREAD

**Issue by cobhimself** [Bernardo-MG/docs-maven-skin#188]
It seems the custom style support as mentioned in the documentation is not working correctly.

The configuration:

```
<custom>
   <skinConfig>
      <customStyle>./css/custom.css</customStyle>
   </skinConfig>
</custom>
```

should add `custom.css` to the style on the page.

I've confirmed the style is copied over correctly but it is never imported in the document.

Will be submitting a PR soon.

**Comment by cobhimself** [comment]
Submitted PR  #189

**Comment by Bernardo-MG** [comment]
Merged the pull request and published as version 2.0.12.
